configuration/storage.mdx

```diff
@@ -310,23 +320,75 @@ The following is an example of how to configure Flipt to leverage this backend t
           region: us-east-1
           bucket: flipt_feature_flags
           poll_interval: "30s"
-          # optional bucket prefix for locating flag state files
+          # optional: bucket prefix for locating flag state files
           prefix: production
-          # for non-AWS hosted S3
+          # optional: for non-AWS hosted S3
           endpoint: http://localhost:9009
     ```
 
   </Tab>
 </Tabs>
 
-In addition to these Flipt configuration parameters, credentials will also be required for Flipt to authenticate with the target object-store.
+<Note>
+  In addition to these Flipt configuration parameters, valid credentials will
+  also be required for Flipt to authenticate with the target object-store.
+</Note>
+
 These should be provided as environment variables to the Flipt server process:
 
 ```bash
 AWS_ACCESS_KEY_ID=...
 AWS_SECRET_ACCESS_KEY=...
 ```
 
+#### Azure Blob Storage
+
+The Azure Blob Storage backend can be configured to serve state from a single container from a target Azure Blob Storage account.
+
+The following is an example of how to configure Flipt to leverage this backend type:
+
+<Tabs>
+  <Tab title="Environment Variable">
+
+    ```bash
+    FLIPT_STORAGE_TYPE="object"
+    FLIPT_STORAGE_OBJECT_TYPE="azblob"
+    FLIPT_STORAGE_OBJECT_AZBLOB_CONTAINER="flipt-feature-flags"
+    FLIPT_STORAGE_OBJECT_AZBLOB_POLL_INTERVAL="1m"
+    # optional: for customizing Azure blob storage endpoint
+    FLIPT_STORAGE_OBJECT_AZBLOB_ENDPOINT=http://localhost:10000
+    ```
+
+  </Tab>
+  <Tab title="Configuration Yaml">
+
+    ```yaml
+    storage:
+      type: object
+      object:
+        type: azblob
+        azblob:
+          container: flipt-feature-flags
+          # optional: for customizing Azure blob storage endpoint
+          endpoint: https//devaccount.blob.core.windows.net
+          poll_interval: "30s"
+    ```
+
+  </Tab>
+</Tabs>
+
+<Note>
+  In addition to these Flipt configuration parameters, valid credentials will
+  also be required for Flipt to authenticate with the target object-store.
+</Note>
+
+These should be provided as environment variables to the Flipt server process:
+
+```bash
+AZURE_STORAGE_ACCOUNT=...
+AZURE_STORAGE_KEY=...
+```
+
```

Comment:
These should be provided as environment variables to the Flipt server process. There are 2 options supported for authentication:
  
  * Using [Azure credentials](https://learn.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication?tabs=bash#2-authenticate-with-azure)
  ```bash
  AZURE_CLIENT_ID=...	#Application ID of an Azure service principal
  AZURE_TENANT_ID=...	#ID of the application's Microsoft Entra tenant
  AZURE_CLIENT_SECRET=...	#Password of the Azure service principal
  ```
  * Using Azure Blob storage [account keys](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal)
  
  ```bash
  AZURE_STORAGE_ACCOUNT=... # account name
  AZURE_STORAGE_KEY=... # secret shared key
  ```

Reply:
Oh man... how do I do suggestion with the code blocks?